packages/terra-date-time-picker/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Unreleased
 4.32.0 - (March 24, 2020)
 ------------------
 ### Changed
+* Updated wdio screenshots due to focus style changes in react-datepicker.
```

Comment:
Move this entry to the unreleased section

Reply:
Moved 64aef1f